### ISSUE TITLE
readme is a hot mess

### ISSUE THREAD

**Issue by aschweigert** [INN/Largo-Sample-Child-Theme#17]
this is all kinds of screwed up right now, what's going on here? 

https://github.com/INN/Largo-Sample-Child-Theme/blob/master/docs/readme.md

**Comment by aschweigert** [comment]
@dryanmedia this might be due to something from that last PR, do you want to take the first crack at resolving these conflicts and cleaning this up?

**Comment by 0aveRyan** [comment]
This should now be cleared up!
https://github.com/INN/Largo-Sample-Child-Theme/blob/master/docs/readme.md

**Comment by aschweigert** [comment]
This is looking better, the image is still a bit confusing I think and there's more to flesh out here but I think it works for now.

**Comment by 0aveRyan** [comment]
@aschweigert Yeah let me get the source file for that in the repo today so that's there. It's just hard because there's a LOT of hierarchy and files to cover. 

while we could make the image _longer_,  there's a max-width of 860px for embedding in the GitHub container and that really limits how we can show linkages because there's no room for white space to draw those lines...maybe @kaeti has an idea? 
- [ ] still vital content we're not showing (javascript among others)
- [ ] confusing partially due to lack of space and overlapping lines in places and unclear branding done out of space necessity
- [ ] still needs work
